-1

Tested at: 24184d22d509997b7cd92a82cde7afda7ab6f887

CMSSW: CMSSW_11_2_X_2020-11-11-2300
SCRAM_ARCH: slc7_aarch64_gcc820
You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-373316/10674/summary.html

I found follow errors while testing this PR

**Failed tests:  Build  HeaderConsistency**

* **Build**:

I found compilation error when building: 

<pre>>> Compiling  /home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/src/StreamerInputSource.cc
>> Compiling  /home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/src/StreamerOutputFile.cc
>> Compiling  /home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/src/StreamerOutputModuleBase.cc
>> Compiling  /home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/src/StreamerOutputModuleCommon.cc
In file included from /home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/src/FRDEventMessage.cc:12:
/home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/interface/FRDEventMessage.h:135:97: error: missing braces around initializer for 'std::__array_traits<unsigned int, 7>::_Type' {aka 'unsigned int [7]'} [-Werror=missing-braces]
                                                                               6 * sizeof(uint32)};
                                                                                                 ^
cc1plus: some warnings being treated as errors
In file included from /home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/interface/DumpTools.h:11,
                 from /home/cmsbld/jenkins_a/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-11-11-2300/src/IOPool/Streamer/src/DumpTools.cc:5:
</pre>

--- a/IOPool/Streamer/interface/FRDEventMessage.h
+++ b/IOPool/Streamer/interface/FRDEventMessage.h
@@ -126,13 +126,13 @@ struct FRDEventHeader_V1 {
 const uint16 FRDEVENT_MASK_ISGENDATA = 1;
 
 constexpr size_t FRDHeaderMaxVersion = 6;
-constexpr std::array<uint32, FRDHeaderMaxVersion + 1> FRDHeaderVersionSize = {0,
-                                                                              2 * sizeof(uint32),
-                                                                              (4 + 1024) * sizeof(uint32),
-                                                                              7 * sizeof(uint32),
-                                                                              8 * sizeof(uint32),
-                                                                              6 * sizeof(uint32),
-                                                                              6 * sizeof(uint32)};
+constexpr std::array<uint32, FRDHeaderMaxVersion + 1> FRDHeaderVersionSize {{0,
+                                                                             2 * sizeof(uint32),
+                                                                             (4 + 1024) * sizeof(uint32),
+                                                                             7 * sizeof(uint32),
+                                                                             8 * sizeof(uint32),
+                                                                             6 * sizeof(uint32),
+                                                                             6 * sizeof(uint32)}};
 
 class FRDEventMsgView {
 public:

--- a/IOPool/Streamer/interface/FRDEventMessage.h
+++ b/IOPool/Streamer/interface/FRDEventMessage.h
@@ -126,13 +126,13 @@ struct FRDEventHeader_V1 {
 const uint16 FRDEVENT_MASK_ISGENDATA = 1;
 
 constexpr size_t FRDHeaderMaxVersion = 6;
-constexpr std::array<uint32, FRDHeaderMaxVersion + 1> FRDHeaderVersionSize {{0,
-                                                                             2 * sizeof(uint32),
-                                                                             (4 + 1024) * sizeof(uint32),
-                                                                             7 * sizeof(uint32),
-                                                                             8 * sizeof(uint32),
-                                                                             6 * sizeof(uint32),
-                                                                             6 * sizeof(uint32)}};
+constexpr std::array<uint32, FRDHeaderMaxVersion + 1> FRDHeaderVersionSize{{0,
+                                                                            2 * sizeof(uint32),
+                                                                            (4 + 1024) * sizeof(uint32),
+                                                                            7 * sizeof(uint32),
+                                                                            8 * sizeof(uint32),
+                                                                            6 * sizeof(uint32),
+                                                                            6 * sizeof(uint32)}};
 
 class FRDEventMsgView {
 public:

